azure-pipelines.yml

```
@@ -50,9 +50,13 @@ jobs:
   container: $[ variables['containerResource'] ]
   steps:
     - script: swift test --parallel
-      displayName: swift test
-    - script: swift test --parallel -Xswiftc -DUSE_UTF8
+      displayName: swift test with UTF16
+      env:
+        YAMS_ENCODING: utf16
```

Nit: I'd refer to encodings in their usual all-caps capitalization whenever possible, with exceptions when it would violate Swift's API naming guidelines (e.g. case members or identifiers).

```suggestion
        YAMS_ENCODING: UTF16
```

Same with the other environment variables specified in this file.